index.js

```diff
@@ -3,9 +3,10 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+counter = 2
```

Comment:
Make sure to use `let` or `const` any time you declare a new variable - in this case you'd want to use `let` because the question asked to make a variable and reassign it:

```js
let counter = 1
counter = 2
```